packages/xchain-cosmos/src/client.ts

```diff
@@ -48,9 +59,18 @@ class Client implements CosmosClient, XChainClient {
    *
    * @throws {"Invalid phrase"} Thrown if the given phase is invalid.
    */
-  constructor({ network = 'testnet', phrase }: XChainClientParams) {
+  constructor({
+    network = 'testnet',
+    phrase,
+    rootDerivationPaths = {
+      mainnet: `44'/118'/0'/0/`,
+      testnet: `44'/118'/1'/0/`,
+    },
+  }: XChainClientParams) {
     this.network = network
-    this.sdkClient = new CosmosSDKClient({ server: this.getClientUrl(), chainId: this.getChainId() })
+    this.rootDerivationPaths = rootDerivationPaths
+    this.sdkClients.set('testnet', TESTNET_SDK)
+    this.sdkClients.set('mainnet', MAINNET_SDK)
```

Comment:
Similar to previous comment: Don't store two instance of `CosmosSDKClient`, just one for `testnet` OR `mainnet`.

Reply:
same as above, this was just to cache the sdkclients to speed up things